README.md

```diff
@@ -115,7 +115,8 @@ Before working on a contribution (a new feature or a fix) make sure you can't fi
 5. Create a new branch with your fix / feature from the **develop** branch.
 6. Check that your code pass the tests / design new unit tests: `python -m unittest`.
 7. Verify your tests coverage with `./coverage.sh` (additionaly you can generate xml report and use VSCode Coverage gutter to identify untested lines with `./coverage.sh xml`).
```

Comment:
Shouldn't we change that to encourage using `./gradlew coverageTest` instead ?

Reply:
+1